default-menu-auto-hide.ks.in

```diff
@@ -0,0 +1,17 @@
+#version=DEVEL
+#test name: default-menu-auto-hide
```

Comment:
Shouldn't the test be called `default-menu-auto-hide-rhel`?

Reply:
Fixed.